Agent/Tickets/Personal Queues.rst

```diff
@@ -0,0 +1,126 @@
+Personal queues
```

Comment:
This file needs to be added to `Agent Panel.rst` in order to be included in the sidebar.